velox/substrait/tests/VeloxSubstraitRoundTripTest.cpp

```diff
@@ -313,6 +323,54 @@ TEST_F(VeloxSubstraitRoundTripTest, notNullLiteral) {
       plan, "SELECT true, 23, 45, 678, 910, 1.23, 4.56, '789'");
 }
 
+TEST_F(VeloxSubstraitRoundTripTest, arrayLiteral) {
```

Comment:
@zhejiangxiaomai I has been a long time since I looked at this PR. I'll need to rebuilt the context. Can this test be simplified?

```
namespace {
core::TypedExprPtr makeConstantExpr(const VectorPtr& vector) {
  return std::make_shared<const core::ConstantTypedExpr>(
      BaseVector::wrapInConstant(1, 0, vector));
}
} // namespace

TEST_F(VeloxSubstraitRoundTripTest, arrayLiteral) {
  auto vectors = makeRowVector(ROW({}), 1);
  auto plan =
      PlanBuilder(pool_.get())
          .values({vectors})
          .addNode([&](std::string id, core::PlanNodePtr input) {
            std::vector<std::string> names = {"a", "b", "c"};
            std::vector<core::TypedExprPtr> expressions = {
                // [1, null]
                makeConstantExpr(
                    makeNullableArrayVector<int64_t>({{1, std::nullopt}})),
                // Empty array: [].
                makeConstantExpr(makeArrayVector<int64_t>({{}})),
                // Nested array: [[1, 2, 3], [4, 5]]
                makeConstantExpr(makeArrayVector(
                    {0}, makeArrayVector<int64_t>({{1, 2, 3}, {4, 5}}))),
            };
            return std::make_shared<core::ProjectNode>(
                id, std::move(names), std::move(expressions), input);
          })
          .planNode();
  assertPlanConversion(
      plan, "SELECT array[1, null], array[], array[array[1,2,3], array[4,5]]");
}
```

Reply:
I will try this // Nested array: [[1, 2, 3], [4, 5]] . 
I spent a lot of time getting to know vector module of velox. Sorry for late update this PR.